### ISSUE TITLE
Inconsistencies in Port summary field labels

### ISSUE THREAD

**Issue by jdberry** [macports/macports-webapp#88]
The field labels in the summary tab are inconsistent:

- Most are followed by colons, but "Homepage" is not.
- Most split words, but "ClosedMaintainer" is camel case. "Homepage" could be spelled with or without the space, so that's probably ok as-is.
- Title Case: probably ok as-is. The "Closed Maintainer" is all capitalized while "Maintained by" doesn't capitalize the "by". That's proper for Title case (https://www.bkacontent.com/how-to-correctly-use-apa-style-title-case/), which wouldn't call for the "by", "on", or "of" to be capitalized.

To summarize, I'd probably suggest the following:

 - Be consistent with the trailing colon.
 - Add a space in Closed Maintainer.
 - Probably leave Homepage as one word.

**Comment by jdberry** [comment]
Another question: I suppose there was a conscious choice put into calling it "closed maintainer" vs "open maintainer". Does "closed maintainer" actually have any more meaning to people than "open maintainer", and is that meaning worth the potential confusion of negating the term we normally use and introducing a new, perhaps equally mystifying term?

full disclosure: I take full blame for coming up with the term "open maintainer" originally ;)

**Comment by mojca** [comment]
> The field labels in the summary tab are inconsistent:
> 
> * Most are followed by colons, but "Homepage" is not.

This one is easy to fix, but maybe we can invite you to submit your first pull request and help us improve the site (more importantly: learn how to help in the future)?

It should be as easy as [editing](https://github.com/macports/macports-webapp/edit/master/app/ports/templates/ports/port-detail/summary.html) the following line: https://github.com/macports/macports-webapp/blob/1af5000856b800c02c48ad71b481041843344171/app/ports/templates/ports/port-detail/summary.html#L7
(and opening a pull request, which probably even happens automatically when you click edit and try to save it). See also https://trac.macports.org/wiki/CommitMessages

> * Most split words, but "ClosedMaintainer" is camel case.

That one should go. As explained here [here](https://docs.google.com/document/d/1TqNHm3NrtLDelWqY7P4pZyTMZDJrlsPymFZklzV2t24/edit) I would actually like to completely replace that row with just a simple lock symbol and maybe a pop-up explaining what this means.

I still believe that we basically failed with design of `openmaintainer` and we should be specifying when a port is NOT `openmaintainer`, rather than vice versa (Port should be `openmaintainer` by default). I still hope we'll fix this in `Portfile`s one day, but until we do, I decided to store the opposite to the database :)

> Another question: I suppose there was a conscious choice put into calling it "closed maintainer" vs "open maintainer". Does "closed maintainer" actually have any more meaning to people than "open maintainer", and is that meaning worth the potential confusion of negating the term we normally use and introducing a new, perhaps equally mystifying term?

I would not expose that word. Just show the lock.

**Comment by mojca** [comment]
PS: I'm sorry. I didn't initially notice that you were here way before I was and that you were the one who originally tried to write this long-expected app, as well as
https://trac.macports.org/wiki/SummerOfCodeArchive#a2008
I haven't seen that original name appear for ages and I thought that the original author (i.e. you) had long disappeared from MacPorts, so I simply didn't associate your github handle ...

**Comment by jdberry** [comment]
> PS: I'm sorry. I didn't initially notice that you were here way before I was

np! I've been pretty out of macports for a while... I just stick my head in every now and again...
